data/campaigns/Heir_To_The_Throne/utils/httt_utils.cfg

```diff
@@ -93,17 +128,19 @@
                                 description= _ "This massive blade was created centuries ago by long-forgotten elvish forgemasters, who imbued the bluish steel with an inner magical fire. Tongues of flame dance on the surface, giving the metal a flawless mirrored finish."
                                 cannot_use_message= _ "Only the leader of an army can wield this sword!"
                                 [filter]
-                                    type=Fighter,Commander,Lord,Princess,Battle Princess,Elvish Captain,Elvish Hero,Elvish Marshal,Elvish Champion,Paladin,Elvish Lord,Elvish High Lord
+                                    type=Fighter,Commander,Lord,Princess,Battle Princess,Elvish Captain,Elvish Hero,Elvish Marshal,Elvish Champion,Elvish Lord,Elvish High Lord
```

Comment:
If you read a few lines lower in the `[effect]`, it specifically says `type=blade`, which means it indeed has no effect on the paladin's lance (which is pierce) or sword (which is arcane). If you want the paladin to be able to get it, you'll need to change the weapon filter in the `[effect]`, but I'm not sure there'd be much benefit to switching arcane to fire.